Doc/library/_interpreters.rst

```diff
@@ -0,0 +1,69 @@
+:mod:`_interpreters` --- Low-level interpreters API
```

Comment:
There should be a warning about the module being private, and/or the API being subject to change without notice, IMHO.

Reply:
Given the not-yet-approved PEP, I think it would be a good idea to use a name that makes it clear that this really is intended as an API for the CPython test suite, rather than for general use.

That would mean going with something like `_xxsubinterpreters` for now.

Reply:
Note: one consequence of that is that redistributors that remove the test suite would likely remove this module as well, and I think that's a desirable outcome: we want this as a learning tool for the further development of PEP 554 as a proposal for 3.8 (and to find and fix bugs in the subinterpreter support in 3.7), not as an inadvertent commitment to a particular API.